agent/tests/parsing/test_lsdyna.py

```diff
@@ -0,0 +1,57 @@
+"""
+Test the LS-Dyna parser
```

Comment:
I think we're missing tests for the exceptions, aren't we?

Reply:
We're testing the 3 possible outputs the license server can return to us: correct data, data with no licenses in use, and unparseable data.
I can't think of other tests case to add here.